src/js/JSONEditor.js

```diff
@@ -44,6 +44,20 @@ var util = require('./util');
  *                               {boolean} sortObjectKeys If true, object keys are
  *                                                        sorted before display.
  *                                                        false by default.
+ *                               {Object[]} toolbarPlugins  Array of custom toolbar
```

Comment:
This docblock is outdated and should be removed entirely.  [API Doc](https://github.com/josdejong/jsoneditor/blob/develop/docs/api.md) should be sufficient for documenting this.